cloud/scope/powervs_machine.go

```diff
@@ -202,6 +202,17 @@ func (m *PowerVSMachineScope) CreateMachine() (*models.PVMInstanceReference, err
 		// TODO need a reasonable wrapped error.
 		return instanceReply, nil
 	}
+
+	if instanceReply == nil {
```

Comment:
this if is not required, if we have reached this step only when instanceReply is nil

Reply:
Yes fixed.